server/services/analyticsService.test.ts

```diff
@@ -7,6 +7,23 @@ import { MockTable, mockAppS3ClientResponse } from '../testData/s3Bucket'
 jest.mock('@aws-sdk/client-s3')
 jest.mock('../data/s3Client')
 
+const prisonLocations = {
+  // TODO: it's not yet entirely clear which graphs should filter locations and which should not
+  MDI: {
+    filteredLocations: ['All', '1', '2', '3', '4', '5', '6', '7', '8', 'SEG', 'TAP'],
+    unfilteredLocations: ['All', '1', '2', '3', '4', '5', '6', '7', '8', 'SEG'],
```

Comment:
these come from 2 different data sources, and since different things are being counted, the complete set of wings/locations may _not_ be referenced in either